app/views/hearings/show.html.haml

```diff
@@ -9,7 +9,15 @@
     = render partial: 'details', locals: { hearing_summary: @prosecution_case.hearing_summaries.find { |each| each.id == @hearing_id },
                                            hearing_day: @hearing_day }
 
-    = render partial: 'attendees', locals: { hearing: @hearing, hearing_details: @hearing.try(:hearing) }
+    - is_prosecution_case = @hearing.try(:hearing)&.prosecution_cases&.any?
+    - if is_prosecution_case
+      = render partial: 'attendees_on_prosecution_cases', locals: { hearing: @hearing,
+                                                                    hearing_details: @hearing.try(:hearing),
+                                                                    prosecution_case: @prosecution_case }
+    - else
+      = render partial: 'attendees', locals: { hearing: @hearing,
```

Comment:
I think there's a strong case for renaming this file to "_attendees_on_court_applications.html.haml" rather than just "attendees"